docs/proposals/hpa/federated-hpa-v2.md

```diff
@@ -0,0 +1,699 @@
+---
+title: Federated HPA
+authors:
+- "@jwcesign" # Authors' GitHub accounts here.
+reviewers:
+- TBD
+approvers:
+- TBD
+
+creation-date: 2023-02-14
+
+---
+
+# Federated HPA
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. 
+
+A good summary is probably at least a paragraph in length.
+-->
+
+Horizontal Pod Autoscaling (HPA) is a popular method for stabilizing applications by adjusting the number of pods based on different metrics. While Karmada simplifies running applications on multiple clusters, it currently lacks HPA support.
```

Comment:
The reason why Karmada is going to introduce FederatedHPA there are some specific use cases that HPA can't handle. (not the `lacks HPA support`).

Reply:
It would be better to briefly describe the pain points of using HPA in a single cluster or the limitations in multi-cluster scenarios.

Reply:
modified, please check again